### PR TITLE
Amazon SNS sender

### PR DESCRIPTION
**Pull request by alex-sysoev**
## Amazon SNS sender

**ISSUE #5** 

In order to be able to send messages directly to SNS topic as one of the existing options I will implement AWS SNS sdk to make it possible.

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -2,6 +2,11 @@ use Mix.Config
...
 config :ex_queue_bus_client,
   queue: "communication_bus_develop",
-  event_handler: ExQueueBusClient.EventHandler
+  event_handler: ExQueueBusClient.EventHandler,
+  sns_client: ExAws.SNS,
+  aws_client: ExAws
 
-import_config "dev.secret.exs"
+config :ex_aws,
+  access_key_id: [System.get_env("AWS_ACCESS_KEY"), :instance_role],
```
Comment (sandinosaso) [inline config/dev.exs]: Why dev.exs uses env variables like this? 
Other PR I saw it was the opposite. System.get_env() was for test and prod envs if I remember correctly

File: config/prod.exs [hunk 2]
```diff
@@ -1,8 +1,11 @@
 use Mix.Config
 
 config :ex_aws,
-  access_key_id:      ["${AWS_ACCESS_KEY}", :instance_role],
-  secret_access_key:  ["${AWS_SECRET_KEY}", :instance_role],
+  access_key_id: ["${AWS_ACCESS_KEY}", :instance_role],
```
Comment (sandinosaso) [inline config/prod.exs:4]: Same comment as above

File: lib/ex_queue_bus_client.ex [hunk 3]
```diff
@@ -11,10 +11,38 @@ defmodule ExQueueBusClient do
     SQS.send_message(queue(), action)
   end
 
+  @doc """
+  AWS SQS queue name to use if SQS is configured.
+  """
+  @spec queue() :: binary
   def queue do
     Application.get_env(:ex_queue_bus_client, :queue) || "probando"
```
Comment (sandinosaso) [inline lib/ex_queue_bus_client.ex:28]: What do you think about removing this "probando" default

**Review by sandinosaso — CHANGES_REQUESTED** [review]
2 minor comments after that I can approve

File: config/dev.exs (hunk 1 above)
Comment (alex-sysoev) [reply config/dev.exs]: because they are compile time accessible.

File: config/prod.exs (hunk 2 above)
Comment (alex-sysoev) [reply config/prod.exs:4]: Because they are runtime accessible.

File: lib/ex_queue_bus_client.ex (hunk 3 above)
Comment (alex-sysoev) [reply lib/ex_queue_bus_client.ex:28]: I agree, but in other issue , because this problem belongs to SQS testing stuff. But it makes sense.

**Review by sandinosaso — APPROVED** [review]
LGTM

**Review by dbarria — APPROVED** [review]

**Pull request merged**